scripts/wave-smoke.sh

```
@@ -193,13 +193,24 @@ status() {
+  # default (V-1/V-5 swapped the unauthenticated root to the J2CL
+  # signed-out shell). Fetch the explicit GWT view to assert the
+  # legacy bootstrap is still reachable.
+  gwt_view_body_file=$(mktemp)
+  gwt_view_status=$(curl -sS --max-time 10 -o "$gwt_view_body_file" -w "%{http_code}" "http://localhost:$PORT/?view=gwt" || true)
+  gwt_view_body=$(cat "$gwt_view_body_file" 2>/dev/null || true)
+  rm -f "$gwt_view_body_file"
+  root_gwt_presence=$([[ "$gwt_view_body" == *'webclient/webclient.nocache.js'* ]] && echo present || echo missing)
 
   root_body_file=$(mktemp)
   root_status=$(curl -sS --max-time 10 -o "$root_body_file" -w "%{http_code}" "http://localhost:$PORT/" || true)
   root_body=$(cat "$root_body_file" 2>/dev/null || true)
```

Shellcheck (0.11.0)

[warning] root_body appears unused. Verify use (or export if used externally).

(SC2034)